Core/src/MagneticField/SolenoidBField.cpp

```diff
@@ -10,6 +10,8 @@
 
 #include "Acts/Utilities/Helpers.hpp"
 
+#include <algorithm>
+#include <boost/exception/exception.hpp>
```

Comment:
I have some doubt concerning the : #include <boost/exception/exception.hpp>, especially since it does not appear anywhere in the Acts code...

Reply:
I think it comes in from the fact that we use specific boost::math methods that could call an exception.

Reply:
I see, it make sense I guess.